lampo-common/src/event/liquidity.rs

```diff
@@ -0,0 +1,56 @@
+use bitcoin::secp256k1::PublicKey;
+use lightning::events::HTLCDestination;
+use lightning::ln::{channelmanager::InterceptId, ChannelId, PaymentHash};
+use lightning_liquidity::{lsps0::ser::RequestId, lsps2::msgs::OpeningFeeParams};
+
+#[derive(Debug, Clone)]
+pub enum LiquidityEvent {
```

Comment:
not sure that this is the correct place where to put the event, but maybe this can be moved later in another crate